File: app/src/main/java/eu/warble/pjapp/data/model/PlatnosciItem.java
package eu.warble.pjapp.data.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class PlatnosciItem{

	@SerializedName("Wplacajacy")
	@Expose
	private String wplacajacy;

	@SerializedName("Kwota")
	@Expose
	private double kwota;

	@SerializedName("TytulWplaty")
	@Expose
	private String tytulWplaty;

	@SerializedName("DataWplaty")
	@Expose
	private String dataWplaty;

	public String getWplacajacy(){
		return wplacajacy;
	}

	public double getKwota(){
		return kwota;
	}

	public String getTytulWplaty(){
		return tytulWplaty;
	}

	public String getDataWplaty(){
		return dataWplaty;
	}

	@Override
 	public String toString(){
		return 
			"PlatnosciItem{" + 
			"wplacajacy = '" + wplacajacy + '\'' + 
			",kwota = '" + kwota + '\'' + 
			",tytulWplaty = '" + tytulWplaty + '\'' + 
			",dataWplaty = '" + dataWplaty + '\'' + 
			"}";
		}
}

File: app/src/main/java/eu/warble/pjappkotlin/data/model/PlatnosciItem.kt
package eu.warble.pjappkotlin.data.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName


data class PlatnosciItem(
        @SerializedName("Wplacajacy")
        @Expose
        val wplacajacy: String?,

        @SerializedName("Kwota")
        @Expose
        val kwota: Double?,

        @SerializedName("TytulWplaty")
        @Expose
        val tytulWplaty: String?,

        @SerializedName("DataWplaty")
        @Expose
        val dataWplaty: String?
)